test: _resolve_analyzer_thresholds_defaults reads user override config, fails on machines with adm_aware_excitation=true

## Summary

`tests/unit/mixing/test_analyze_mix_issues.py::test_resolve_analyzer_thresholds_defaults` calls `_resolve_analyzer_thresholds()` directly without isolating from the user's local config. The resolver invokes `tools.mixing.mix_tracks.load_mix_presets()`, which deep-merges `~/bitwize-music/overrides/mix-presets.yaml` over the bundled defaults. If a contributor's override sets `defaults.analyzer.adm_aware_excitation: true`, the test fails:

```
assert adm_aware is False
E   assert True is False
tests/unit/mixing/test_analyze_mix_issues.py:24
```

CI is unaffected because the runner has no user overrides directory.

## Discovered

While verifying PR #359 locally (`make check` against the dependabot dep bump). 3767/3768 tests passed — only this isolation issue surfaced. The dep bump itself was unrelated and merged cleanly.

## Suggested fix

Make the test independent of host config — options:

1. **monkeypatch the loader**: `monkeypatch.setattr("handlers.processing.mixing.load_mix_presets", lambda: {"defaults": {"analyzer": {}}})` so the resolver hits the fallback path.
2. **monkeypatch `_get_overrides_path`** to return a `tmp_path` so no user override file is found.
3. **Point `BITWIZE_MUSIC_CONFIG` (or equivalent) at a temp config** that has no `paths.overrides`, in a session-scoped fixture for the whole `tests/unit/mixing/` package.

Option (1) is the smallest change and directly tests the resolver's contract.

## Repro

```bash
mkdir -p ~/bitwize-music/overrides
cat > ~/bitwize-music/overrides/mix-presets.yaml <<'YAML'
defaults:
  analyzer:
    adm_aware_excitation: true
YAML
make test
```

--- a/tests/unit/mixing/test_analyze_mix_issues.py
+++ b/tests/unit/mixing/test_analyze_mix_issues.py
@@ -15,9 +15,15 @@ if str(SERVER_DIR) not in sys.path:
     sys.path.insert(0, str(SERVER_DIR))
 
 
-def test_resolve_analyzer_thresholds_defaults():
+def test_resolve_analyzer_thresholds_defaults(monkeypatch):
     """With no preset overrides, resolver returns (0.10, 0.25, False)."""
+    import tools.mixing.mix_tracks as mix_tracks
     from handlers.processing.mixing import _resolve_analyzer_thresholds
+
+    # Stub load_mix_presets so the test doesn't read the host's
+    # ~/bitwize-music/overrides/mix-presets.yaml — see #360.
+    monkeypatch.setattr(mix_tracks, "load_mix_presets", lambda: {"defaults": {}})
+
     dark, harsh, adm_aware = _resolve_analyzer_thresholds()
     assert dark == pytest.approx(0.10)
     assert harsh == pytest.approx(0.25)

